k8s/environments/ptl/cluster-00-overlay/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../../release/admin/kube-slack
+- ../../../release/admin/kured
+- ../../../release/admin/secrets-csi-driver
+- ../../../namespaces/azure-devops
+- ../../../namespaces/admin/aad-pod-identity
+- ../../../namespaces/admin/traefik2
+- ../../../namespaces/kube-system/aad-pod-identity
+- ../../../namespaces/monitoring/kube-prometheus-stack
+- ../../../namespaces/jenkins
```

Comment:
if you aren't adding the jenkins patch now I would add this in a follow up PR as it won't work without it

```suggestion
```